diff --git a/Makefile b/Makefile
index 101d59b..2b53e7a 100644
--- a/Makefile
+++ b/Makefile
@@ -37,17 +37,17 @@ default: $(version_h)
 
 BKPT_VER=$(shell cat versions | grep BACKPORTS_RELEASE_TAG | cut -d "_" -f 8 | cut -d "\"" -f 1 | cut -d "-" -f 1 2>/dev/null || echo 1)
 
-BASE_KER_VER = $(shell cat $(KLIB_BUILD)/include/generated/autoconf.h | grep CONFIG_VERSION_SIGNATURE | cut -d '"' -f 2 | cut -d " " -f 2 | cut -d "-" -f 1,2 | cut -d '.' -f 1-3 2> /dev/null)
+BASE_KER_VER = $(shell cat $(KLIB_BUILD)/version | cut -d "-" -f 1,2 | cut -d '.' -f 1-3 2> /dev/null)
 
-KERN_VER = $(shell cat $(KLIB_BUILD)/include/generated/autoconf.h | grep CONFIG_VERSION_SIGNATURE | cut -d '"' -f 2 | cut -d " " -f 2 | cut -d "-" -f 1 | cut -d '.' -f 1,2 2> /dev/null)
+KERN_VER = $(shell cat $(KLIB_BUILD)/version | cut -d "-" -f 1 | cut -d '.' -f 1,2 2> /dev/null)
 
 # DII_TAG is extracted from DII_KERNEL_TAG, which is auto genereated from base kernel source. Tagging is needed
 # for decoding this, Sample in the version file DII_KERNEL_TAG="PROD_DG1_200828.0"
 DII_TAG=$(shell cat versions | grep DII_KERNEL_TAG | cut -f 2 -d "\"" | cut -d "_" -f 2 2>/dev/null || echo 1)
 
 KER_VER := $(shell cat versions | grep BASE_KERNEL_NAME_22.04 | cut -d "\"" -f 2 | cut -d "-" -f 2-|sed "s/-/./g" 2>/dev/null || echo 1)
-UBUNTU_BACKPORT_MAJOR = $(shell cat $(KLIB_BUILD)/include/generated/autoconf.h | grep CONFIG_VERSION_SIGNATURE | cut -d '-' -f 2 | cut -d '.' -f 1 2> /dev/null)
-UBUNTU_BACKPORT_MINOR = $(shell cat $(KLIB_BUILD)/include/generated/autoconf.h | grep CONFIG_VERSION_SIGNATURE | cut -d '-' -f 2 | cut -d '.' -f 2 2> /dev/null)
+UBUNTU_BACKPORT_MAJOR = 22
+UBUNTU_BACKPORT_MINOR = 4
 
 ifeq ($(UBUNTU_BACKPORT_MAJOR),)
 	UBUNTU_BACKPORT_MAJOR = 0;
